README.md

```diff
@@ -1,15 +1,62 @@
-[![TLA+ Build Status](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
+Overview
+--------
+[![](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
 
-For more information, visit http://lamport.azurewebsites.net/tla/tla.html
+This repository hosts the core TLA⁺ command line interface (CLI) tools and the Toolbox integrated development environment (IDE).
+Its development is managed by the [TLA⁺ Foundation](https://foundation.tlapl.us/).
+See http://tlapl.us for more information about TLA⁺ itself.
+For the TLA⁺ proof manager, see https://github.com/tlaplus/tlapm.
+
+Versioned releases can be found on the [Releases](https://github.com/tlaplus/tlaplus/releases) page.
+Currently, every commit to the master branch is built & uploaded to the [1.8.0 Clarke pre-release](https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0).
+If you want the latest fixes & features you can use that pre-release.
+
+The CLI tools and Toolbox IDE are both written in Java.
+The CLI tool source code is in [tlatools/org.lamport.tlatools](./tlatools/org.lamport.tlatools).
+The Toolbox IDE is based on [Eclipse Platform](https://github.com/eclipse-platform/eclipse.platform) and is in [toolbox](./toolbox).
+
+Build & Test CLI Tools
+----------------------
+Install the following dependencies to your path:
+ * [OpenJDK](https://openjdk.org/) version 11+
+ * [Apache Ant](https://ant.apache.org/) version 1.9.8+
+
+Clone this repo & open a shell in its root, then run:
+```bash
+cd tlatools/org.lamport.tlatools
+ant -f customBuild.xml compile compile-test dist # Builds tla2tools.jar
+java -cp dist/tla2tools.jar tlc2.TLC test-model/pcal/Bakery.tla # Runs an example
+ant -f customBuild.xml test # Runs unit tests (optional)
+```
+
+The unit tests should run for 4-10 minutes depending on how fast your computer is, and succeed.
```

Comment:
I would remove the expected duration. This is the kind of brittle number that depends on many factors and is _certain_ to increase as we improve test coverage.